### PR TITLE
Modify documentation (dataservice-read)

### PR DESCRIPTION
**Pull request by HalynaDumych**
Modify documentation in the public files (dataservice-read/include)
and two private files that contain Doxygen descriptions.

Relates-to: OLPEDGE-856

Signed-off-by: Halyna Dumych <ext-halyna.dumych@here.com>

### CODE REVIEW

File: olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/ConsumerProperties.h [hunk 1]
```diff
@@ -23,33 +23,67 @@
 #include <string>
 #include <vector>
 
-#include <boost/optional.hpp>
 #include <olp/dataservice/read/DataServiceReadApi.h>
+#include <boost/optional.hpp>
```
Comment (mykolamalik) [inline olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/ConsumerProperties.h]: According to clang Google style "Your project's .h files" have to be **after** "Other libraries' .h files". So include files order was ok and you should not change the order. Please refer to https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes for more details.

Comment (HalynaDumych) [reply olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/ConsumerProperties.h]: Thank you!
Fixed

File: olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/ConsumerProperties.h [hunk 2]
```diff
@@ -31,25 +31,59 @@ namespace dataservice {
 namespace read {
 
 /**
- * @brief Represents one stream layer consumer config entry.
+ * @brief Represents a consumer configuration entry of one stream layer.
  *
- * An entry can be one of: string, int32, bool.
+ * Examples: `string`, `int32`, `bool`.
```
Comment (andescu) [inline olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/ConsumerProperties.h]: The previous comment denoted that the accepted variable types are string, int32 or bool.
Your comment implies you should use these values as string input. Or are you also referring to variable data types?

Comment (HalynaDumych) [reply olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/ConsumerProperties.h]: Changed

**Pull request merged**